rclcpp/include/rclcpp/strategies/allocator_memory_strategy.hpp

```diff
@@ -108,34 +108,34 @@ class AllocatorMemoryStrategy : public memory_strategy::MemoryStrategy
     client_handles_.clear();
   }
 
-  void revalidate_handles()
+  void remove_null_handles()
   {
-    {
-      VectorRebind<void *> temp;
-      for (auto & subscriber_handle : subscriber_handles_) {
-        if (subscriber_handle) {
-          temp.push_back(subscriber_handle);
-        }
-      }
-      subscriber_handles_.swap(temp);
+    auto it = std::remove(
+      subscriber_handles_.begin(),
+      subscriber_handles_.end(),
+      nullptr
+      );
+    if (it != subscriber_handles_.end()) {
+      subscriber_handles_.erase(it, subscriber_handles_.end());
```

Comment:
I've cleaned it up a bit. How do you think the indentation looks? 6271cb58973bb121cfeec22d195208d2022ba682